<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketTableForm</class>
 <widget class="QWidget" name="TicketTableForm">
  <layout class="QVBoxLayout" name="verticalLayout">

  <item>
   <widget class="QLabel" name="labelDone">
    <property name="text">
     <string>Текущие задачи</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold; font-size: 14px;</string>
    </property>
   </widget>
  </item>

   <!-- Основная таблица -->
   <item>
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">
QTableView {
    border: none;
    gridline-color: black;
}

QHeaderView::section {
    border: 1px solid black;
    background: #f0f0f0;
    font-weight: bold;
}

QTableCornerButton::section {
    border: none;
}
      </string>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>

   <!-- Отступ (фиксированная высота) -->
   <item>
    <widget class="QWidget" name="spacerWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
    </widget>
   </item>

   <!-- Заголовок завершённых -->
   <item>
    <widget class="QLabel" name="labelDone">
     <property name="text">
      <string>Завершённые задачи</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 14px;</string>
     </property>
    </widget>
   </item>

   <!-- Таблица завершённых -->
   <item>
    <widget class="QTableView" name="doneTableView">
     <property name="styleSheet">
      <string notr="true">
QTableView {
    border: none;
    gridline-color: black;
}

QHeaderView::section {
    border: 1px solid black;
    background: #f0f0f0;
    font-weight: bold;
}

QTableCornerButton::section {
    border: none;
}
      </string>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
